jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/FusekiServer.java

```diff
@@ -1704,5 +1713,28 @@ private static void applyLocalhost(Server server) {
                 }
             }
         }
+
+        private static Map<String,String> parseCORSConfigFile(String filename) {
+            try {
```

Comment:
Is using a JSON file overkill here?  Wondering if we might not be better just taking in a Java Properties file as then the whole method reduces to:

```java
try (FileInputStream input = new FileInputStream(filename)) {
  Properties properties = new Properties();
  properties.load(input);
  // Convert Properties into Map
  return map;
}
```

Reply:
No reason - happy to change.